docs/demos/filters/min_max_filter.jl

```diff
@@ -0,0 +1,56 @@
+# Topic : Max Min Filters
+# reference paper : https://core.ac.uk/download/pdf/194053536.pdf
+
+
+# In this tutorial we see how can we can effectively use max and min filter to distinguish 
+# between ramp("smooth") and ripple("texture") edges of B&W image.We will using the mapwindow function of
+# ImageFiltering.jl package.
+
+# Note : If you are using the terminal use imshow(<Image var. name>) to view the image. 
+
+using Images, ImageFiltering
+
+# We download an image from the internet and convert it into Grayscale image.
+# To work with your own image, use :
+# img_path = "<FULL PATH TO THE IMAGE>"
+
+img_path = download("https://i.pinimg.com/originals/01/bc/8d/01bc8d82a3e2b4fa869f478479b97a3f.png")
+Img = Gray.(load(img_path));       
```

Comment:
Variables in julia should typically be lower-case. Matrices are an exception due to the legacy of math, otherwise it's better to follow the typical stylistic conventions:
- https://docs.julialang.org/en/v1/manual/style-guide/#Use-naming-conventions-consistent-with-Julia-base/
- https://github.com/invenia/BlueStyle

I'll only comment on this here, but it's relevant for all your variables.

Reply:
I am aware of the lowercase variable naming syntax, I was facing problem for the min and max which Julia assumes to be functions.

Reply:
How about `min_out`/`img_out_min`, `max_out`/`img_out_max`, etc?

Reply:
Yes, there are many other names to choose from, so I'd rather follow the stylistic convention. In addition to the ones @johnnychen94  mentioned, `imgmin` and `imgmax` seem OK too.